### PR TITLE
Remove support for C++/Javascript version of evm2wasm

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@chfast after fixing this, can we merge this for 0.2.1?

**Comment by chfast** [comment]
> @chfast after fixing this, can we merge this for 0.2.1?

Yes, why not.

**Comment by chfast** [comment]
Merge https://github.com/ewasm/hera/pull/453 and rebase this one.

**Comment by axic** [comment]
Either depends on #375 or all evm2wasm tests need to be removed.

**Comment by chfast** [comment]
I'm for disabling evm2wasm tests and re-enable them when runevm is ready.

**Comment by axic** [comment]
Build is failing due to https://github.com/ethereum/evmc/issues/172

**Comment by axic** [comment]
@chfast even if you close https://github.com/ethereum/evmc/issues/172 this is still broken and needs to be addressed :)

**Comment by chfast** [comment]
> @chfast even if you close [ethereum/evmc#172](https://github.com/ethereum/evmc/issues/172) this is still broken and needs to be addressed :)

No kidding.

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,18 +12,16 @@ include(CableToolchains)
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
+option(EVMC_TESTING "Build EVMC tests and test tools" OFF)
```
Comment (axic) [inline CMakeLists.txt]: Isn't this a foreign key? `${EVMC_TESTING}` instead of `OFF`?

Comment (chfast) [reply CMakeLists.txt]: No. It's defined before EVMC defines it. I think this line can be dropped actually.

Comment (axic) [reply CMakeLists.txt]: I'll just merge this now.

Comment (chfast) [reply CMakeLists.txt]: I will remove the line.

**Pull request merged**